cpp/src/rolling/grouped_rolling.cu

```diff
@@ -371,26 +373,27 @@ std::unique_ptr<column> time_range_window_ASC(column_view const& input,
 /// Each group in the input timestamp column must be sorted,
 /// with null values clustered at either the start or the end of each group.
 /// If there are no nulls for any given group, (nulls_begin, nulls_end) == (0,0).
```

Comment:
Looking at the call sites of this function I see the following comment ( ` For input of n groups, group_offsets has n+1 values. Drop the last value using a span` ). 
I think we should update the above comment to explain that `group_offsets` is now inclusive on both ends, instead of exclusive on the backend.

Reply:
I decided to revert this due to potential confusion.